specification.md

```diff
@@ -250,6 +277,46 @@ In OpenTracing we force `SpanContext` instances to be **immutable** in order to
 
 This is modeled in different ways depending on the language, but semantically the caller should be able to efficiently iterate through all baggage items in one pass given a `SpanContext` instance.
 
+### `ScopeManager`
+
+The `ScopeManager` interface sets and retrieves the active `Span`, working along the `Scope` container interface.
+Formally, it has the following capabilities:
+
+#### Activate a `Span`
+
+Prior to setting a provided `Span` as active for the current thread or execution unit, the previously active instance needs to be stashed away, so it can be restored afterwards.
+
+Required parameters
+
+- **span**, the `Span` instance to be set as active.
+- **finish Span on close**, a boolean value specifying whether the `Span` should be `Finish`ed upon being deactivated through its container `Scope` being `Close`d.
+
+**Returns** a `Scope` instance containing the newly activated `Span`.
+
+#### Retrieve the active `Span`
+
+There should be no parameters.
+
+**Returns** a `Scope` instance containing the active `Span`, or else `null` if there's none for the current thread or execution unit.
```

Comment:
It would be good to resolve https://github.com/opentracing/opentracing-java/issues/267  before adding this to the spec

Reply:
I'd say it's more a related effort - as part of this very PR, we need to discuss what is described in #267 (as well as https://github.com/opentracing/opentracing-ruby/pull/36).

But yes, in practical terms, we need to figure those two issues above before finally merging this PR 😉